webapp/utils/websocket_client.jsx

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import WebSocketClient from 'mattermost/websocket_client.jsx';
+
+var WebClient = new WebSocketClient();
+export default WebClient;
```

Comment:
We need to instantiate it somewhere so it's nicer to do it here just like we do for the other client.jsx. As for the setting the var first I just copied our other one :P